File: java/producer-json/src/main/java/com/github/alopatin/kafka_learning/java/producer_json/model/iot/LocationMeasurementJson.java
package com.github.alopatin.kafka_learning.java.producer_json.model.iot;

import com.fasterxml.jackson.annotation.JsonProperty;

public class LocationMeasurementJson {
    @JsonProperty
    String location;

    @JsonProperty
    double measurement;

    public LocationMeasurementJson(String location, double measurement) {
        this.location = location;
        this.measurement = measurement;
    }

    @Override
    public String toString() {
        return "LocationMeasurementJson{" + "location='" + location + "'" + ", measurement=" + measurement + "}";
    }
}

File: scala/producer-json/src/main/scala/com/github/alopatin/kafka_learning/scala/producer_json/model/iot/LocationMeasurementJson.scala
package com.github.alopatin.kafka_learning.scala.producer_json.model.iot

import com.fasterxml.jackson.annotation.JsonProperty


class LocationMeasurementJson(
                               _location: String,
                               _measurement: Double
                                  ) {
  @JsonProperty("location")
  var location: String = _location

  @JsonProperty("measurement")
  var measurement: Double = _measurement
}
